knowledge_base/job_read_secret/databricks.yml

```
@@ -0,0 +1,47 @@
+bundle:
+  name: job-read-secret-example
+
+# workspace:
+#   host: https://myworkspace.cloud.databricks.com
+
+resources:
+  secret_scopes:
+    my_secret_scope:
+      name: secrets-scope-1
+      permissions:
+        - level: CAN_VIEW
```

```suggestion
        - level: READ
```

CAN_VIEW and CAN_MANAGE are not defined for secret scopes, only: `READ | WRITE | MANAGE`

---

We map `CAN_VIEW` to `READ` and `CAN_MANAGE` to `MANAGE` permissions in dabs cli. A possible oversight here is that we do not map `CAN_WRITE` to `WRITE`